cmd/nginx/main.go

```diff
@@ -119,13 +119,22 @@ func main() {
 	mux := http.NewServeMux()
 	go registerHandlers(conf.EnableProfiling, conf.ListenPorts.Health, ngx, mux)
 
-	lc, err := collector.NewInstance(conf.Namespace, class.IngressClass, 8000)
+	ngxStatusPath := "/nginx_status"
```

Comment:
@fmejia97 this makes no sense now. Please remove this variable and parameter from NewNGINXStatusCollector